staging/src/k8s.io/kube-scheduler/config/v1beta1/types_pluginargs.go

```diff
@@ -239,3 +243,35 @@ type NodeAffinityArgs struct {
 	// +optional
 	AddedAffinity *corev1.NodeAffinity `json:"addedAffinity,omitempty"`
 }
+
+type ScoringStrategyType string
+
+const (
+	LeastAllocated           ScoringStrategyType = "LeastAllocated"
+	MostAllocated            ScoringStrategyType = "MostAllocated"
+	RequestedToCapacityRatio ScoringStrategyType = "RequestedToCapacityRatio"
+)
+
+// ScoringStrategy define ScoringStrategyType for node resource plugin
+type ScoringStrategy struct {
+	// Type selects which strategy to run.
+	Type ScoringStrategyType `json:"type,omitempty"`
```

Comment:
oh man, this should be a reference. We wanted to revert the change, in the original one those were references.